xBrokerConnector/src/main/java/dev/midas/xbc/XBCApp.java

```diff
@@ -0,0 +1,28 @@
+package dev.midas.xbc;
+
+import dev.midas.xbc.config.ConfigManager;
+import dev.midas.xbc.stream.ListeningEngine;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class XBCApp implements CommandLineRunner{
+
+    @Autowired
+    private final ListeningEngine listening;
+
+    public XBCApp(ListeningEngine listening) {
+        this.listening = listening;
+    }
```

Comment:
I don't think we need the constructor if we use the `@Autowired`, but may be wrong